server/ComputeSuperpixelFeatures/ComputeSuperpixelFeatures.py

```diff
@@ -0,0 +1,387 @@
+from skimage.measure import regionprops
+from skimage.segmentation import slic
+from keras.models import Model, load_model
+
+import os
+import sys
+import json
+import time
+
+import numpy as np
+import dask
+import h5py
+
+import histomicstk.preprocessing.color_normalization as htk_cnorm
+import histomicstk.preprocessing.color_deconvolution as htk_cdeconv
+import histomicstk.utils as htk_utils
+
+import large_image
+
+from ctk_cli import CLIArgumentParser
+
+import logging
+logging.basicConfig(level=logging.CRITICAL)
+
+sys.path.append(os.path.normpath(os.path.join(os.path.dirname(__file__), '..')))
+from cli_common import utils as cli_utils  # noqa
+
+
+def compute_input_tensors(slide_path, tile_position, args, **it_kwargs):
```

Comment:
Maybe rename to `compute_super_pixel_data`

Reply:
Renamed to compute_superpixel_data